spacy/cli/package.py

```diff
@@ -430,6 +446,18 @@ def _format_label_scheme(data: Dict[str, Any]) -> str:
     return md.text
 
 
+def _is_permitted_package_name(package_name: str) -> bool:
+    permitted_match = re.search(
+        r"^([A-Z0-9]|[A-Z0-9][A-Z0-9._-]*[A-Z0-9])$", package_name, re.IGNORECASE
+    )
+    return permitted_match is not None
+
+
+def _contains_multiple_underscores(package_name: str) -> bool:
+    permitted_match = re.search(r"_{2,}", package_name, re.IGNORECASE)
+    return permitted_match is not None
```

Comment:
```suggestion
    return "__" in package_name
```

You can probably just use this directly instead of in a separate function.

Reply:
Function removed and used directly in more recent commit.